src/metadata/EntityMetadata.ts

```diff
@@ -485,14 +486,14 @@ export class EntityMetadata {
     /**
      * Creates a new entity.
      */
-    create(): any {
+    create(queryRunner: QueryRunner): any {
```

Comment:
`queryRunner` should be optional. Don't pass it using `createQueryRunner()` into the method in the places **where you don't need to do it**.